tests/reporting_utils.py

```diff
@@ -96,26 +96,36 @@ def finalizeFork(fixture, market, universe, finalizeByMigration = True):
 
     if (finalizeByMigration):
         # Tester 0 moves more than 50% of REP
```

Comment:
Is there a test for users to migrate after fork? They should not get the bonus.

Reply:
Yea! That's one of the tests I added in this. Note that if finalizeByMigration isn't true when finalizing a fork it will not add the bonus in the final migration balance check